common/artifacts/907_wc_artifacts_rings_generic.txt

```diff
@@ -7,10 +7,9 @@ ring_5_diplomacy = {
 	# active = { }
 
 	vassal_opinion = 2
-	same_religion_opinion = 4
-	church_opinion = 8
-	sex_appeal_opinion = 20
-
+	liege_opinion = 4
+	child_opinion = 15
+	dynasty_opinion = 8
 	monthly_character_prestige = 0.5
```

Comment:
So, I noticed that the rings give 2 times less prestige, but not other modifiers. Is it intended?

Reply:
Values of all modifiers for rings are halved

Reply:
Oh, I see now.